plugins/generator-1.14.4/forge-1.14.4/procedures/item_show_totemlike_animation.java.ftl

```diff
@@ -0,0 +1,3 @@
+if (!world.getWorld().isRemote) {
```

Comment:
Remove the negation, since remote means client-side.

Reply:
Done!